model/src/lib.rs

```diff
@@ -132,6 +132,13 @@ impl DomainSeparator {
     }
 }
 
+#[derive(Clone, Debug, serde::Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SolvableOrders {
```

Comment:
Curious about two identical `SolvableOrders` struct. Could we just have one?

Reply:
The reason there are multiple is that they serve different purposes. The one in model is for serialization, the one in SolvableOrdersCache also contains update time which model doesn't.